content/millennium/dstu2/medications/medication-statement.md

```diff
@@ -7,6 +7,39 @@ title: MedicationStatement | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The Medication Statement Resource provides a snapshot in time of known medications taken by the patient now or in the past reported by either the patient, significant other or a provider. Future orders are not returned. Documented historical/past/home medications are commonly captured when taking the patient's medical history. Prescriptions without documented compliance are Intended, since we may not know if the patient is actively taking the medication or has filled the prescription. Medications are assumed to be Taken unless documented otherwise. 
+<br/><br/>
+References to implicitRules and modifierExtensions are NOT supported and will fail a Create or Update request.
+<br/><br/>
+The following fields are returned if valued:
```

Comment:
Should we also indicate that supportingInformation will be populated with a reference to MedicationOrder?